<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string comment="TESTING!!!" extracomment="TESTING!!!!"/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string>Don't have an account? Register now!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="registerNotImplementedLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="authFailedLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordTextbox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameTextbox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="loadingLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>270</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>usernameTextbox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>loginButton</zorder>
  <zorder>registerButton</zorder>
  <zorder>registerNotImplementedLabel</zorder>
  <zorder>authFailedLabel</zorder>
  <zorder>passwordTextbox</zorder>
  <zorder>loadingLabel</zorder>
 </widget>
 <tabstops>
  <tabstop>usernameTextbox</tabstop>
  <tabstop>passwordTextbox</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>registerButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
